### ISSUE TITLE
Bug Web Service Api Prestashop when create or update the quantity of a product

### ISSUE THREAD

**Issue by maxime18dev** [PrestaShop/PrestaShop#24520]
#### Describe the bug

In the Web Service Api of Prestashop, I can't modify the quantity of a product. I discovered that I had to go through stock available to be able to modify the quantity of a product but once I added quantities on a product, the quantity was added correctly (visible in the back office) but when I go to the ps_product table, the value in the "quantity" column is always '0'. I don't know if it's a bug or if it's not the right way to modify the quantity of a product with the web service api. 

#### Expected behavior

Modify the value of the 'quantity' column in the table 'ps_product'.

#### Steps to reproduce

1. 'Post' a product with the web service api
```
$createdProductXml = $webService->add([
    'resource' => 'products',
    'postXml' => $productBlankXml->asXML()
]);
```
2. Update the quantity of the product with 'stock availables'
```
$stockAvailableXml = $webService->get([
        'resource' => 'stock_availables',
        'id' => $idStockAvailable
    ]);

    $dataStockAvailable = $stockAvailableXml->stock_available->children();

    $dataStockAvailable->quantity=999;

    $updatedXml = $webService->edit([
        'resource'=>'stock_availables',
        'id'=>$idStockAvailable,
        'putXml' => $stockAvailableXml->asXML()
    ]);
```
3. In the back office, when you go in catalog>the product u added
you can see the quantity correctly but when you go in the database, the value in the column 'quantity' of the table 'ps_product' is still '0'.

**Screenshots**

![image](https://user-images.githubusercontent.com/71233149/118459077-ab2f9180-b6fb-11eb-99a1-a32fb48124c8.png)

![image](https://user-images.githubusercontent.com/71233149/118459328-f0ec5a00-b6fb-11eb-8278-1ea6e4e302b5.png)

![image](https://user-images.githubusercontent.com/71233149/118459635-390b7c80-b6fc-11eb-98b5-6ba55a46f8dc.png)

#### Additional information

* PrestaShop version: 1.7.7.2
* PHP version: 7.3.21

**Comment by hibatallahAouadni** [comment]
Hello @maxime18dev 

We are aware of this issue, it is already in our debug backlog. This issue is a duplicate of #18953. 
To be informed when it's fixed, please subscribe to the issue mentioned above.

Thanks!

**Comment by hibatallahAouadni** [comment]
Duplicate of #18953

**Comment by maxime18dev** [comment]
> Hello @maxime18dev
> 
> We are aware of this issue, it is already in our debug backlog. This issue is a duplicate of #18953.
> To be informed when it's fixed, please subscribe to the issue mentioned above.
> 
> Thanks!

Thanks for your answer, when do you think this issue will be fixed ? because the #18953 is opened since the 3 may 2020

@hibatallahAouadni

**Comment by hibatallahAouadni** [comment]
Hello @maxime18dev 

Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you. 

If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you
